src/components/LoginComponent.js

```diff
@@ -132,11 +141,7 @@ const LoginComponent = ({ notifyLogIn, displayPopup, loggedIn, route }) => {
       {!state.login && (
         <div>
           <TextField
-            error={
-              !state.login &&
-              state.passTwo !== '' &&
-              state.passTwo !== state.pass
-            }
+            error={!state.login && state.passTwo !== '' && state.passTwo !== state.pass}
```

Comment:
When I originally made my part, the error prop completely slipped my mind. I might add it in with the final round of testing but wouldn't be too big of a deal